.github/workflows/docs.yml

```diff
@@ -7,11 +7,7 @@ on:
     branches: [ master ] # build docs as additional test on PRs to main
   workflow_dispatch:
     branches: [ '*' ] # allow manual trigger for all branches
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
-  
```

Comment:
Why did you remove concurrency?

Reply:
ups, this was not intended. Should be fixed now!